docs/pages/workflow/continuous-native-generation.mdx

```diff
@@ -25,91 +26,254 @@ Expo as a framework enables **Continuous Native Generation** by combining the fo
+
+**With Prebuild** upgrading is much closer to upgrading a pure JavaScript application. Bump the versions in your **package.json** and regenerate the native project.
+
+</Collapsible>
+
+<Collapsible summary="How does Prebuild simplify cross-platform configuration?">
+
+Cross-platform configurations such as the app icon, name, splash screen, and so on must be implemented manually in native code. These are often implemented very differently across platforms.
+
+**With Prebuild** cross-platform configurations are handled at the config plugin level, and the developer only needs to set a single value like `"icon": "./icon.png"` to have all icon generation taken care of.
+
+</Collapsible>
+
+<Collapsible summary="How to manage dependency side-effects with Prebuild?">
+
+Many complex native packages require additional setup beyond installing and [autolinking](/more/glossary-of-terms#autolinking). For example, a camera library requires permission settings to be added **AndroidManifest.xml** for Android and **Info.plist** for iOS. This additional setup can be considered a configuration side effect of a package. Pasting the required side effect code into your project's native files can lead to difficult native compilation errors, and it's also code that you now own and maintain.
+
+**With Prebuild** library authors, who know how to configure their library better than anyone, can create a testable and versioned script called a [config plugin](/config-plugins/introduction/), to automate adding the required configuration side effects for their library. This means library side effects can be more expressive, powerful, and stable. For native code side effects, we also provide [Android Lifecycle Listeners](/modules/android-lifecycle-listeners) and [AppDelegate Subscribers](/modules/appdelegate-subscribers) which come standard in the default [prebuild template](#templates).
+
+</Collapsible>
+
+<Collapsible summary="How does Prebuild help with orphaned code?">
+
+When you uninstall a package you have to be certain you removed all of the side effects required to make that package work. If you miss anything, it leads to orphaned code that you cannot trace back to any particular package, this code builds up and makes your project harder to understand and maintain.
+
+**With Prebuild** the only side effect is the [config plugin](/config-plugins/introduction/) in a project's Expo config (**app.json**), which will throw an error when the corresponding node module has been uninstalled, meaning a lot less orphaned configuration.
+
+</Collapsible>
+
+<Collapsible summary="When Prebuild might not be the right fit for a project?">
+
+Here are some reasons Expo Prebuild might **not** be the right fit for a particular project:
+
+#### React Native versioning
```

Comment:
Lately, we've released in conjunction with RN releases and not necessarily quarterly. However, this is a moving target, with flexibility in recent SDK's to use multiple versions. So, not entirely sure what should be in this section; you might consider just striking the entire thing.

Reply:
Done, removed this section.